react-native/architecture/white-label.mo.md

```diff
@@ -0,0 +1,117 @@
+# [MO] How to implement white-label on a React Native app _(~<Time> min)_
+
+## Owner: [Maxime Sraïki](https://github.com/sraikimaxime)
+
+## Prerequisites
+
+- Generate a new React Native Project using rn-generator-toolbox
+- Have all of your styles (at least the colors) in a centralized appstyle.js file
+- Have your colors named in an agnostic way (primary > myClientBlue)
```

Comment:
To me it's not clear which one I should choose. For example should I choose "primary" or "deepBlue" or "BAMBlue"?